WPMedia/fusion-news-theme-blocks
# PEN-672: Article Landing Page : Footer > Headings such as 'Terms of Use', 'Contact Us' are identified as list items by screen readers

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
Applied recommended fix (see ticket) for screen reader issue where list heading was being identified as a list item. Essentially just moved the heading out of the list element.

## Jira Ticket
- [PEN-672](https://arcpublishing.atlassian.net/browse/PEN-672)

## Acceptance Criteria
![image](https://user-images.githubusercontent.com/26662906/107051944-b08f4e80-6792-11eb-9332-0ee06159acbd.png)

## Test Steps
I was not able to find a way to test this on my machine since the software used to report this issue is Windows-only (JAWS 2019). I attempted to use Apple VoiceOver to replicate the issue but for me, it doesn't speak the count of each footer list (simply identifies each link in the list correctly). After discussing with Sara, we are going to let the accessibility testing firm handle the testing for the accessibility fix.

To test for any styling regressions, open any page with a footer block and make sure everything looks good and the header elements are still 14px font size. It should look exactly the same as it does in the Core Components environment.

## Effect Of Changes
### Before
Using JAWS 2019 screen reader, the header of each footer list was being counted as a list item when reading the "count" of list items to the user

### After
Using JAWS 2019 screen reader, the header of each footer list is **no longer** being counted as a list item when reading the "count" of list items to the user

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist 
_The reviewer of the PR should copy-paste this template into the review comments on review._
